docs/alerting/monitors.md

```diff
@@ -41,6 +41,41 @@ Destination | A reusable location for an action, such as Amazon Chime, Slack, or
 
    This information is stored in plain text in the Elasticsearch cluster. We will improve this design in the future, but for now, the encoded credentials (which are neither encrypted nor hashed) might be visible to other Elasticsearch users.
 
+###  Email as a destination
+To receive an alert notification as an email, you must set up **Email** as the destination **Type** for the alert, and configure **Sender** email addresses and **Recipients** or email groups. You can do this using **Manage senders** and **Manage email groups**. 
+
+#### Manage senders
+Manage senders allows you to configure and manage **Sender** email addresses. Senders are email accounts from where the alert notification is sent to different recipients or email groups. Unless you configure at least one sender account in the settings, the **Sender** dropdown list will be empty. 
```

Comment:
First and last sentence can both go.

Reply:
If the first sentence goes then I would be leading with a definition and not a use case. And your initial comment for the previous para was -  "Lead with a use case, not a definition." So I've just deleted the last sentence.